tests/test_pysoot.py

```diff
@@ -95,17 +91,17 @@ def test_android1():
     apk = os.path.join(test_samples_folder, "android1.apk")
     lifter = Lifter(apk, input_format="apk", android_sdk=sdk_path)
     subc = lifter.soot_wrapper.getSubclassesOf("java.lang.Object")
-    nose.tools.assert_in("com.example.antoniob.android1.MainActivity", subc)
+    assert "com.example.antoniob.android1.MainActivity" in subc
     main_activity = lifter.classes["com.example.antoniob.android1.MainActivity"]
 
     tstr = str(main_activity)
     tokens = ["onCreate", "ANDROID1", "TAG", "Random", "android.os.Bundle", "34387"]
     for t in tokens:
-        nose.tools.assert_in(t, tstr)
+        assert t in tstr
     # l.debug("client std\n%s" % lifter.soot_wrapper.get_client_std())
+test_android1.speed = "slow"
```

Comment:
Here and below, please add a blank newline between the function declaration and the attribute assignment

Reply:
Done